Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -757,6 +757,9 @@ size_t Calendar::day_of_week_index(String const& day_name)
 
```

Comment:
The commit message could start with something a bit more expressive about the issue like:
Calendar: Accept the Calendar notifications only